src/CMakeLists.txt

```diff
@@ -79,6 +79,7 @@ swig_lib(NAME      openroad_swig
          SCRIPTS   ${OPENSTA_HOME}/tcl/Util.tcl
                    OpenRoad.tcl
                    ${ICEWALL_HOME}/src/ICeWall.tcl
+                   ${ICEWALL_HOME}/src/place_cell.tcl
```

Comment:
why is this a separate file?

Reply:
I made it a separate file because it's entirely independent of all the rest of the ICeWall code - essentially its just a more "user friendly" API to the odb:: function calls. 
Whilst we need this functionality, I'm not convinced that it should reside within ICeWall. Potentially this allows it to be moved somewhere else.

Reply:
@Colin-Holehouse I don't think this belongs here. It should probably be moved to ORD, but because its behavior can also add cells, I think you are better off integrating it into ICeWall.

Reply:
I've added place_cell.tcl onto the end of ICeWall.tcl